Lib/test/test_compile.py

```diff
@@ -995,6 +995,20 @@ def continue_in_while():
             self.assertEqual('RETURN_CONST', opcodes[1].opname)
             self.assertEqual(None, opcodes[1].argval)
 
+    @support.cpython_only
+    def test_return_ifexp_const_optimization(self):
+        # gh-121246: Check that the CPython-specific peephole optimizer would
+        # properly convert pairs of LOAD_CONST + RETURN_VALUE into
+        # RETURN_CONST even if they are separated by NOP or in different
+        # blocks of instructions as a result of prior optimizations for
+        # returning different constants from a ternary operation
```

Comment:
Typically here we just write "see gh-121346" in a comment. The discussion should be on the issue.